include/rocksdb/db.h

```diff
@@ -110,6 +110,16 @@ struct RangePtr {
   RangePtr(const Slice* s, const Slice* l) : start(s), limit(l) { }
 };
 
+struct IngestExternalFileArg {
+  ColumnFamilyHandle* column_family;
```

Comment:
When I'm trying the API yesterday I found out a problem - if there are duplicate column families in the args, the API would fail with a confusing error message <sst_file> already exists. Adding a pass in MyRocks grouping by column families fixed that issue. In terms of API design, I think it'd be great if this API can issue a better error if duplicate column families is found. Another option is to group it within the API but there are addition complications if the options are different (or you need to check it as well).

Reply:
+1 for better error reporting. If there are duplicate column families, the API function should return an accurate error.